crates/cli/src/opts/ethereum.rs

```diff
@@ -140,6 +140,46 @@ impl EtherscanOpts {
     }
 }
 
+#[derive(Clone, Debug, Default, Serialize, Parser)]
+pub struct OKLinkOpts {
+    /// The OKLink (or equivalent) API key.
+    #[arg(long = "oklink-api-key", env = "OKLINK_API_KEY")]
+    #[serde(rename = "oklink_api_key", skip_serializing_if = "Option::is_none")]
+    pub apikey: Option<String>,
+}
```

Comment:
I'm skeptical that we need this, I believe all of this already exists, and can be solved with the --verifier argument instead?